<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarInfo</class>
 <widget class="QDialog" name="ModificarInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModificarInfo{
	border-image: url(:/Img/FONDO.jfif);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDITAR PERFIL</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="Editar_buttonBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-width: 0px;
border-radius: 8px;
background: rgb(254, 0, 0, 50);}

QPushButton:hover{
	background-color: rgb(154,158,218);
    border-radius: 9px;
	color: black;
    selection-color: white;  
    padding: 2px 4px;}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>381</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
background-image: url(:/Img/F1.jfif);
border-radius: 8px;
}

QLineEdit{
background-color:transparent;
border:none;
border-bottom: 1px solid;
border-color: rgb(56, 143, 147);}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="NOmbre_lineEdit">
       <property name="placeholderText">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ApellidoP_lineEdit_2">
       <property name="placeholderText">
        <string>Apellido Paterno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ApellidoM_lineEdit">
       <property name="placeholderText">
        <string>Apellido Materno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NumeroT_lineEdit">
       <property name="placeholderText">
        <string>Número Telefonico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Calle_lineEdit_2">
       <property name="placeholderText">
        <string>Calle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NumeroC_lineEdit">
       <property name="placeholderText">
        <string>Número</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mail_lineEdit">
       <property name="placeholderText">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DATOS PERSONALES</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>271</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{
background-image: url(:/Img/F1.jfif);
border-radius: 8px;
}

QLineEdit{
background-color:transparent;
border:none;
border-bottom: 1px solid;
border-color: rgb(56, 143, 147);}</string>
   </property>
   <widget class="QLineEdit" name="Matricula_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Matricula</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Tipo de vehiculo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Tipo_Coche_comboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DATOS DEL VEHICULO</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Img/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
